rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "nightly-2021-10-16"
+channel = "nightly-2021-11-10"
```

Comment:
Why is the toolchain that old? To match sgx?

Reply:
Sad story..